docs/docs/user/references/editor-integration.md

```diff
@@ -0,0 +1,22 @@
+# Editor integration
+
+KPOps provides JSON schemas that enable autocompletion and validation for some of the files that the user must work with.
+
+## Supported files
+
+- [`pipeline.yaml`](/user/references/components)
+- [`config.yaml`](/user/references/config)
+
+## Usage
+
+1. Install the
+[yaml-language-server](https://github.com/redhat-developer/yaml-language-server#clients) in your editor of choice. (requires LSP support)
+2. Configure the extension with the settings below.
+
+???+ "`settings.json`"
```

Comment:
is this working with the `+`?

I noticed other embeds have this prelude
```
??? "`pipeline.yaml`"
```

Reply:
I checked it locally and it's working

Reply:
the `+` indicates that the block should be expanded by default which I thought suits the page better.